src/extractors/htmlcharacterextractor.py

```diff
@@ -0,0 +1,36 @@
+import json
+from pathlib import Path
+import requests
+from typing import List
+
+from .characterfactory import Character
+from .extractor import Extractor
+
+class HtmlCharacterExtractor(Extractor):
+    __characters: List[Character]
+
+    def __init__(self):
+        self.__characters = []
+
+    def __fetch_data(self) -> None:
+        response: requests.Response = requests.get(
+            "https://html.spec.whatwg.org/entities.json", timeout=120
+        )
+        entities = json.loads(response.text)
+        for key in entities:
+            if key.endswith(';') and key != "&NewLine;":
+                name = key[1:-1]
+                char = ''.join(list(map(chr, entities[key]['codepoints'])))
+                self.__characters.append(Character(char, name))
+
+    def __write_to_file(self, target: Path) -> None:
+        if len(self.__characters) == 0:
```

Comment:
I don't think this is necessary - if there are no characters, we'll just write an empty file. (Or, maybe, we should do a similar error handling in all extractors).

Reply:
I was following the lead of NerdFontExtractor - but indeed seems find to drop